tools/cli/commands.go

```diff
@@ -85,6 +85,10 @@ const (
 	FlagEarliestTimeWithAlias      = FlagEarliestTime + ", et"
 	FlagLatestTime                 = "latest_time"
 	FlagLatestTimeWithAlias        = FlagLatestTime + ", lt"
+	FlagPrintEventVersion          = "print_event_version"
+	FlagPrintEventVersionWithAlias = FlagPrintEventVersion + ", pev"
+	FlagPrintFullyDetail           = "print_full"
+	FlagPrintFullyDetailWithAlias  = FlagPrintFullyDetail + ", full"
```

Comment:
Maybe change alias "full" to "pf" to align with all other Print flags?

Reply:
Yeah good idea.